contracts/MakerOrders.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.4.18;
+
+
+import "./Withdrawable.sol";
+
+
+contract MakerOrders is Withdrawable {
+    struct FreeOrders {
+        uint32 firstOrderId;
+        uint32 numOrders; //max is 256
+        uint8 maxOrdersReached;
+        uint256 takenBitmap;
+    }
+
+    //each maker will have orders that will be reused.
+    mapping(address => FreeOrders) public makerOrdersSell;
+    mapping(address => FreeOrders) public makerOrdersBuy;
```

Comment:
thought the mapping should be in extending contract. its not used here anyway. and this layer don't have to be aware of sell buy diff.

thats what we talked...

Reply:
Done.